content/guides/models/automations/_index.md

```diff
@@ -3,7 +3,341 @@ menu:
   default:
     identifier: automations
     parent: w-b-models
-title: Automations
+title: Create and manage automations
 weight: 4
 ---
 
+This page describes _automations_ in W&B and shows how to create and manage them. Create an automation to trigger workflow steps, such as automated model testing and deployment, based on an event in W&B, such as when an [artifact]({{< relref "/guides/core/artifacts" >}}) or a [registered model is changed]({{< relref "/guides/models/registry/" >}}), or based on the results of a [run]({{< relref "/guides/models/track/runs/manage-runs.md" >}})
+
+An automation defines the [event scopes and types]({{< relref "#event-scopes-and-types" >}}) to watch for and the [action]({{< relref "#event-actions" >}}) to take when the event occurs, such as running a webhook or posting to a Slack channel.
+
+## Event scopes and types
+
+An automation's triggering event depends on the automation's scope.
+
+### Registry
+For a [registered model]({{< relref "/guides/models/registry/">}}), you can configure an automation to run on these events:
```

Comment:
I believe there's no restriction to just the registered model types, it's any datatype in any collection.

I'm pretty sure the term "registered model" is not a thing anymore, it's an artifact (!) of when that was the only type of registry, but I might be wrong.